recipes-bsp/bootfiles/rpi-cmdline.bb

```diff
@@ -29,6 +33,13 @@ CMDLINE_LOGO ?= '${@oe.utils.conditional("DISABLE_RPI_BOOT_LOGO", "1", "logo.nol
 # to enable kernel debugging.
 CMDLINE_DEBUG ?= ""
 
+# Add RNDIS capabilities (must be after rootwait)
+# example: 
+# CMDLINE_RNDIS = "modules-load=dwc2,g_ether g_ether.host_addr=<some MAC 
+# address> g_ether.dev_addr=<some MAC address>"
+# if the MAC addresses are omitted, random values will be used
+CMDLINE_RNDIS ?= ""
+
```

Comment:
this seems unrelated, can this be removed.

Reply:
I'm a bit confused here. I didn't touch the code of any file. I just added the headers.